backend/functions/lib/configs/authorization-configuration.ts

```diff
@@ -3,13 +3,16 @@ import { AuthorizationConfig } from '../adapters/types/authorization';
 
 dotenv.config();
 
+const doMockAuth = process.env.MOCK_AUTH === 'true';
```

Comment:
Why bother setting up another env when we can leverage provider and do

`const doMockAuth = process.env.CAMS_LOGIN_PROVIDER === 'mock';`

Reply:
We could have done it that way too. There was code already in place to look for the MOCK_AUTH env var so we built off that.